### PR TITLE
Update project cards

### PR DESCRIPTION
**Pull request by enoqueJonas**
### Additions:
- Added a popup window

## Improvements 
- Updated the project cards section so it loads data dynamically
- Updated the popup window so it loads data dynamically

### CODE REVIEW

File: script/script.js [hunk 1]
```diff
@@ -7,13 +7,362 @@ const navItemPortfolio = document.querySelector('#nav-item-portfolio');
...
+      // Creating the main container that holdds the popup
+      projectsPopupDiv.className = 'projects-popup';
+      projectsPopupDiv.classList.toggle('active-popup');
+      projectsPopupDiv.appendChild(popupProjectHeaderDiv);
+      projectsPopupDiv.appendChild(projectPopupImg);
+      projectsPopupDiv.appendChild(projectPopupDetails);
+      projectCardsSection.appendChild(projectsPopupDiv);
+    }
+  });
+}
```
Comment (m05e5) [inline script/script.js]: - Hi. As mentioned in the requirement, each popup is supposed to have details about a specific card but from your UI all the projects have popups with details about only one project 

image

- [ ] please Make sure each popup has details about the clicked project
- [ ] Currently, the popup is not scrollable and it's overflowing the screen. kindly fix it
- [ ] Please to improve UI ill recommend adding a blur or a background color with opacity when the popup shows

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @enoqueJonas , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check comments below

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style-mobile-screen.css [hunk 2]
```diff
@@ -44,10 +44,6 @@ a {
   bottom: 0;
```
Comment (HAMMAS-SALEEM) [inline css/style-mobile-screen.css:44]: image

- [ ] You can see that your design is not very much aligned in center in different screen sizes. Kindly make it similar to the Figma template :blush:
- [ ] Your popup is showing the same data when we click on any button. Your popup should show data dynamically from your array of objects. Kindly fix it :blush:
- [ ]  Previous reviewer asked you to add opacity, blur :thinking: You have added opacity but you didn't add blur. Kindly add it :+1: to make your design look similar to the Figma template :+1:

Note: You can view this [pull request and javascript](https://github.com/HAMMAS-SALEEM/Portfolio/pull/16/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28f) file to get an idea how you can achieve dynamic data from array of objects to popup window :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @enoqueJonas :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters
:heavy_check_mark: Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @enoqueJonas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**